src/gnssro/gnssro_bufr2ioda.py

```diff
@@ -180,14 +184,15 @@ def get_obs_data(bufr, profile_meta_data, record_number=None):
     bang_conf = codes_get_array(bufr, 'percentConfidence')[1:krepfac[0]+1]
```

Comment:
Should this be checked as one of the QC flags so that it can be consistent with the Fortran code?